packages/create-svelte/templates/default/src/app.d.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="@sveltejs/kit" />
+
+declare namespace App {
```

Comment:
We should add a link to the docs here so people know what this is about

Reply:
done